css/danti.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
```

Comment:
The `css` folder should contain css files only - you can keep this HTML file in the root of the project, along with `index.html`. 

I'd also suggest naming `danti.css` and `danti.html` `store.css` and `store.html` to make it obvious what they provide :+1: